### ISSUE TITLE
catmull slider error when calculating ur

### ISSUE THREAD

**Issue by CCleanerShot** [circleguard/circlecore#160]
_Circleguard works on any other user loadable with all investigations_

When I added a user loadable for user ID **11068425** , I moved to it for investigations "Unstable Rate", "Snaps", and "Frametime". When I ran it, an error had occured. 

```
[ERROR] 2020/09/09 04:27:51.0626 Error while running circlecore. Please report this to the developers through discord or github.
 (circleguard_gui, main_tab.py:600)
Traceback (most recent call last):
  File "gui\main_tab.py", line 553, in run_circleguard
  File "site-packages\circleguard\circleguard.py", line 137, in run
  File "site-packages\circleguard\investigator.py", line 50, in investigate
  File "site-packages\circleguard\investigator.py", line 73, in ur
  File "site-packages\circleguard\investigator.py", line 326, in hits
  File "site-packages\slider\beatmap.py", line 1478, in hit_objects
  File "site-packages\slider\beatmap.py", line 1545, in _resolve_stacking
  File "site-packages\slider\curve.py", line 311, in __call__
NotImplementedError: catmull positions not supported yet
```

I retested immediately with a different user (11160462) , then back to this user ID. No luck. I've been doing this for about a dozen+ accounts so far, but this is the 1st continuous error i'm running across

**Comment by tybug** [comment]
Unstable Rate investigations can fail if the replay was played on an old beatmap. Old beatmaps used catmull sliders, and the beatmap parsing library we use (https://github.com/llllllllll/slider) doesn't have support for them yet. Quit a few people are running into this though so I'm going to prioritize fixing it.

Same issue seen here: https://github.com/circleguard/circlevis/issues/2

I'll leave this one open for tracking as well.

**Comment by tybug** [comment]
closed in v5.0.1 (underlying fix here: https://github.com/llllllllll/slider/pull/74)
